src/commcare_cloud/ansible/roles/couchdb2/handlers/main.yml

```diff
@@ -9,3 +9,11 @@
 - name: stop couchdb2
   service: name=couchdb2 state=stopped
 
+- name: config changes
+  pause:
+    prompt: >-
+      The configuration changes won't be applied until CouchDB is restarted
+      (please follow your internal documentation to do that safely), or the
+      runtime configuration modified manually (e.g. via API calls) if
+      applicable. Press Enter to acknowledge and continue
```

Comment:
I think we don't want to happen on first time install, right? Any way to prevent that?

Reply:
This is only presented when run with the 'couchdb_extras' tag, which seems unlikely for a first time install, or are you thinking about a different scenario?

Reply:
Oh I see I hadn't digested what the line below meant. That makes sense, all good